extensions/wikia/AdEngine/js/video/player/ui/replayOverlay.js

```diff
@@ -0,0 +1,50 @@
+/*global define*/
+define('ext.wikia.adEngine.video.player.ui.replayOverlay', [
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (doc, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.player.ui.replayOverlay',
+		replayOverlayClass = 'replay-overlay';
+
+	function add(video) {
+		var overlay = doc.createElement('div');
+
+		overlay.classList.add(replayOverlayClass);
+		overlay.addEventListener('click', function () {
```

Comment:
Shall we store it in Kikimora as a new event for VUAP? //cc @fraszczakszymon @nefretete